M2/Macaulay2/packages/Macaulay2Doc/overview3.m2

```diff
@@ -368,6 +369,21 @@ document {
      To see the version of the library your copy of Macaulay2 is linked with, examine the variable ", TO "version", "."
      }
 
+doc ///
+  Key
+    "msolve"
```

Comment:
Shouldn't this be documented in the package, since M2 isn't linked with msolve?

Reply:
My impression was that all the programs that are potentially built and distributed with Macaulay2 (e.g., 4ti2, TOPCOM, etc.) should be documented in `Macaulay2Doc`.

Reply:
I could be wrong, but I think almost nobody installs M2 in a way that comes bundled with the programs anymore (e.g. through brew, ubuntu, other package managers, ...). The rare exception is someone building M2 from scratch on a server that has nothing installed, and I don't think whether or not this documentation exists makes a big difference to them. Perhaps the page "Copyright and license" should instead link to the page in the appropriate package, e.g. instead of listing `msolve` as a `Subnode`, it should link to "MSolve :: msolve" or something like that.

Reply:
But perhaps this is out of scope for this PR. I'll mark this as resolved and we can move them all out later.